controllers/chat/group/leaveGroupMember.js

```diff
@@ -46,17 +49,18 @@ const leaveGroupMember = async (req, res) => {
         is_from_prepopulated: true,
         system_user: userId,
         isSystem: true,
-        members: members
-      });
+        members: all_members
+      },
+      {
+        skip_push: true
+      }
+    );
 
-      channelResponse = updatedChannel;
+    channelResponse = updatedChannel;
 
-      channelApiResponse.channel.name = newChannelName;
-      channelApiResponse.channel.better_channel_member = betterChannelMember;
-      channelApiResponse.channel.better_channel_member_object = betterChannelMemberObject;
-    } catch (e) {
-      console.log(e);
-    }
+    channelApiResponse.channel.name = newChannelName;
+    channelApiResponse.channel.better_channel_member = betterChannelMember;
+    channelApiResponse.channel.better_channel_member_object = betterChannelMemberObject;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These lines are modifying the `channelApiResponse` object directly. This may lead to unintended side effects if this object is used elsewhere in the code. Consider creating a copy of the object before modifying it.

```diff
-    channelApiResponse.channel.name = newChannelName;
-    channelApiResponse.channel.better_channel_member = betterChannelMember;
-    channelApiResponse.channel.better_channel_member_object = betterChannelMemberObject;
+    let updatedChannelApiResponse = {...channelApiResponse};
+    updatedChannelApiResponse.channel.name = newChannelName;
+    updatedChannelApiResponse.channel.better_channel_member = betterChannelMember;
+    updatedChannelApiResponse.channel.better_channel_member_object = betterChannelMemberObject;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->